AutomationCourse_HomeWork/Task_1.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace AutomationCourse_HomeWork
+{
+    class Task_1
+    {
+        static void Main()
+        {
+            Console.WriteLine("Please add three arguments for the futher operations:");
+            Console.Write("x = ");
+            double x = Convert.ToInt32(Console.ReadLine());
```

Comment:
variable got type double, but we are doing cast to int, why?

Reply:
Fixed.